text/0000-lts.md

```diff
@@ -0,0 +1,164 @@
+on supporting particular *versions*, focus on supporting particular *release
+channels*. That is, a library may guarantee that it will compile on any of the
+stable release channels, determining how old the version of Rust their users
+can be on with a guarantee of compatibility.
+
+We recommend that libraries consider dropping support for a release channel to
+be a breaking change, but that dropping support for any particular version of
+Rust to be a non-breaking change. 
+
+If a library does not support a particular release channel, effectively all of
+the libraries that depend on *it* also do not support that channel. For this
+reason, we strongly recommend that so-called "core libraries" - those on which
+many other libraries depend - adopt a practice of supporting all three stable
+release channels. Examples of core libraries include libc and serde.
+
+For libraries that do not believe they are "core libraries," we make no
+particular recommendation about which channels to support. There is always a
+trade off between providing the best API for users on stable as soon as
+possible vs supporting users who choose to stay on older versions of Rust, and
+each library should make a judgment call regarding the trade off in their
+particular circumstances.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Current policy is that the core team decides if a bugfix justifies a patch
+release to the stable channel. This policy will remain the same for patches to
+the lts and lts-prior channels.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+This increases the maintenance burden on the release and infrastructure teams
+by adding two additional release channels.
+
+This also muddies our recommendation that users upgrade to the latest stable,
+which we have worked hard to make very easy to do. In particular, we should not
+allow the existence of lts releases to justify reducing our effort to identify
+and resolve stable-to-stable regressions.
+
+Ultimately, users will continue not to upgrade to the latest stable, and
+libraries will continue to want to support users on some older versions of
+Rust. Providing a policy allows a better consensus to be formed around what is
+supported and what isn't.
+
+# Rationale and alternatives
+[alternatives]: #alternatives
+
+There are many ways to decide how to provide LTS support. We have chosen to
+extend our existing channels mechanism by adding more channels, rather than
+constructing a parallel LTS mechanism.
+
+The main knobs within that context are in regard to when we choose to make lts
+releases and how many lts channels we support. Our choice to support two lts
+channels, released every 4 stable releases, is inspired by the ember project's
+LTS system.
+
+# Prior art
+[prior-art]: #prior-art
+
+Many different projects offer an LTS release. Our most direct inspiration was
+the ember project. The ember project's LTS support is described well by both
```

Comment:
This does surprise me. I would think that some of the users most interested in LTS releases are more industrial applications including embedded devices. I concur with others that something called "LTS" is surprising to only provide support for 6 months / 1 years.

Reply:
@Centril Would you prefer it be longer than 1 year, or would you prefer it being called something else?

Reply:
I have no preference. But *if* we are going to call it "LTS", then I think it should be longer than 1 year.
On the other hand, if it is longer than 1 year, then even more bandwidth will be required of the people doing the back-porting of bugfixes.

Reply:
Time to invent "MTS" (Medium Term Support) releases.

Reply:
"long" is a relative term, and in this case, 1 year vs 6 weeks